iq/iq.go

```diff
@@ -87,67 +95,64 @@ type Configuration struct {
 	Version       string
 	OSSIndexUser  string
 	OSSIndexToken string
+	PollInterval  time.Duration
 }
 
-type IQServerError struct {
-	Err     error
-	Message string
-}
-
-func (i *IQServerError) Error() string {
-	if i.Err != nil {
-		return fmt.Sprintf("An error occurred: %s, err: %s", i.Message, i.Err.Error())
+func (i *IQServer) New(logger *logrus.Logger, options Options) *IQServer {
+	if options.PollInterval == 0 {
+		options.PollInterval = 1 * time.Second
```

Comment:
Maybe add a log here that we are defaulting the value to something since you passed in 0??